pkg/models/errors.go

```diff
@@ -23,6 +23,7 @@ var (
 	ErrNetworkOverlaps                       = errors.New("cluster network overlaps")
 	ErrImmutableTwoFactorDelete              = errors.New("twoFactorDelete field is immutable")
 	ErrImmutableCloudProviderSettings        = errors.New("cloudProviderSettings are immutable")
+	ErrNotEmptyCSRs                          = errors.New("certificate creation allowed only if the user is created on the specific cluster")
```

Comment:
certificate creation allowed only if the user is created on the specific cluster -> certificate creation allowed only if the user is already created on the cluster, wdyt?

Reply:
Because of cert creation flow. It's need the cluster ID